src/routers/dummyRoute.ts

```diff
@@ -1,5 +1,5 @@
 import { Router } from 'express';
-import { dummyRequest } from '../controllers/dummyController';
+ import dummyRequest from '../controllers/dummyController';
 
```

Comment:
Please don't change someone else's structure code here you have imported it as default while it's was not default imports